eng/scripts/build.ps1

```diff
@@ -1,6 +1,7 @@
 #Requires -Version 7.0
 param([string]$filter, [switch]$clean, [switch]$vet, [switch]$generate, [switch]$skipBuild, [switch]$cleanGenerated, [switch]$format, [switch]$tidy, [string]$config = "autorest.md", [string]$outputFolder)
 
+. $PSScriptRoot/meta_generation.ps1
```

Comment:
I have a question about this one: what does this used for?

Reply:
It is used for old _meta.json config converting. As it is deprecated, I've removed related logic.